src/renderer/components/wallet/AssetDetails.tsx

```diff
@@ -139,6 +141,14 @@ const AssetDetails: React.FC<Props> = (props: Props): JSX.Element => {
     )
   }, [menuItems])
 
+  const onChangePagination = useCallback(
+    (pageNo) => {
+      loadSelectedAssetTxsHandler({ limit: MAX_PAGES, offset: (pageNo - 1) * MAX_PAGES })
```

Comment:
im really can not understand why offset equals `(pageNo -  1) * MAX_PAGES`. could you pls clarify ? 
seems `MAX_PAGES` is a constant which tells how much elements we allowed to show per page ?

Reply:
Good point. `MAX_PAGES` is misleading here, it should be `MAX_ROWS` or `MAX_ITEMS`. I'm going to change it ...